modules/images/dominant-color/hooks.php

```diff
@@ -244,3 +244,20 @@ function dominant_color_add_inline_style() {
 	wp_add_inline_style( $handle, $custom_css );
 }
 add_filter( 'wp_enqueue_scripts', 'dominant_color_add_inline_style' );
+
+/**
+ * Displays the HTML generator tag for the Dominant Color Images plugin.
+ *
+ * See {@see 'wp_head'}.
+ *
+ * @since n.e.x.t
+ */
+function dominant_color_render_generator() {
+	if (
+		defined( 'DOMINANT_COLOR_IMAGES_VERSION' ) &&
+		! str_starts_with( DOMINANT_COLOR_IMAGES_VERSION, 'Performance Lab ' )
```

Comment:
@felixarntz checking if this is supposed to be a false conditional check? I see further on in `load.php` you _do_ define`DOMINANT_COLOR_IMAGES_VERSION ` starting with `'Performance Lab '`, so I'm struggling to see why there is a false check against this. I feel I ma be missing some context here though.

Reply:
It seems to be that it will always be prefixed with `Performance Lab `.

Reply:
@10upsimon We set this meta tag for standalone plugin. When workflow run `npm run build-plugins` [script will changes](https://github.com/WordPress/performance/blob/trunk/bin/plugin/commands/build-plugins.js#L89-L94) the constant value `define( 'DOMINANT_COLOR_IMAGES_VERSION', 'Performance Lab ' . PERFLAB_VERSION );` to `define( 'DOMINANT_COLOR_IMAGES_VERSION', '1.0.0' );` which we define in `plugins.json` file.

Reply:
Thank you @mukeshpanchal27 this now makes total sense. Resolving this comment thread.